atlasdb-commons/src/main/java/com/palantir/common/base/Throwables.java

```diff
@@ -103,10 +112,8 @@ private static RuntimeException createPalantirRuntimeException(Throwable ex) {
     private static RuntimeException createPalantirRuntimeException(String newMessage, Throwable ex) {
         if (ex instanceof InterruptedException || ex instanceof InterruptedIOException) {
             Thread.currentThread().interrupt();
-            return new PalantirInterruptedException(newMessage, ex);
```

Comment:
Are we aware that it changes the exception type? `PalantirInterruptedException`-s are now `PalantirRuntimeException`-s.

Reply:
Does this break any tests for the large internal product? The change wasn't intentional to be honest, but AtlasDB has moved away from wrapping exceptions from CassandraKVS or Timelock service into `PalantirRuntimeException`s. So, technically there are breaking changes in these codepaths. Would like to understand the implications of these changes, if any. Also, a revert is welcome or I can pick it up tomorrow.

Reply:
Yes, large internal product is sad about this. If you can revert this will be great, I pinged Jeremy about it. Will cc you as well.